droopler.install

```diff
@@ -410,6 +410,7 @@ function droopler_update_8103() {
       'd_p_text_paged',
       'd_p_text_with_bckg',
       'd_p_tiles',
+      'd_p_media',
```

Comment:
Wstawmy to do najnowszego hooka `droopler_update_8107()`, bo 8103 nie odpali się jeśli ktoś będzie aktualizował z beta1/beta2